tests/functional/test_compilers.py

```diff
@@ -1,3 +1,13 @@
 def test_get_imports(project, compilers):
     # See ape-solidity for better tests
     assert not compilers.get_imports(project.source_paths)
+
+
+def test_missing_compilers_without_source_files(project):
```

Comment:
test to make sure that if no missing compilers it also works fine - there are only json contracts in project